frontend/packages/process-editor/src/components/ConfigViewerPanel/ConfigViewerPanel.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { StudioDisplayTile, StudioSectionHeader } from '@studio/components';
+import { useBpmnContext } from '../../contexts/BpmnContext';
+import { ConfigIcon } from '../ConfigPanel/ConfigContent/ConfigIcon';
+import { getConfigTitleHelpTextKey, getConfigTitleKey } from '../../utils/configPanelUtils';
+import { useTranslation } from 'react-i18next';
+import { Alert, Heading, Paragraph } from '@digdir/design-system-react';
+import { ConfigSurface } from '../ConfigSurface/ConfigSurface';
+
+export const ConfigViewerPanel = (): React.ReactElement => {
+  return (
+    <ConfigSurface>
+      <ConfigViewerPanelContent />
+    </ConfigSurface>
+  );
+};
+
+export const ConfigViewerPanelContent = (): React.ReactElement => {
+  const { t } = useTranslation();
+  const { bpmnDetails } = useBpmnContext();
+
+  if (bpmnDetails === null || !bpmnDetails.taskType) {
+    return <ChooseElementToViewAlert />;
+  }
+
+  const configTitle = t(getConfigTitleKey(bpmnDetails.taskType));
+  const configHeaderHelpText =
+    bpmnDetails?.taskType && t(getConfigTitleHelpTextKey(bpmnDetails.taskType));
+
+  const propertiesToDisplay: Array<{ label: string; value: string }> = [
+    {
+      label: t('process_editor.configuration_view_panel_id_label'),
+      value: bpmnDetails.id,
+    },
+    {
+      label: t('process_editor.configuration_view_panel_name_label'),
+      value: bpmnDetails.name,
+    },
+  ];
```

Comment:
Does it make sense placing this in a utils so we can reuse it here and in ConfigContent.tsx?

Reply:
No, not really. Because this is just viewable properties that are not changeable. 😊